moviepy/video/VideoClip.py

```diff
@@ -496,8 +496,8 @@ def write_gif(
         loop : int, optional
           Repeat the clip using ``loop`` iterations in the resulting GIF.
 
-        progress_bar
-          If True, displays a progress bar
+        logger
+          Either ``"bar"`` for progress bar or ``None`` or any Proglog logger.
```

Comment:
Either/or suggests two options, so users may get confused about if this describes three, or if it's indeed two and there's a letter missing ("or any Proglog logger" vs. "_for_ any Proglog logger"; the latter would follow the "x for a" pattern in the first half of the sentence).

Could you reword this to make it clearer if there are two or three options?